doc/whats_new/v0.23.rst

```diff
@@ -56,3 +56,9 @@ Changelog
 
 - |Efficiency| :class:`preprocessing.OneHotEncoder` is now faster at
   transforming. :pr:`15762` by `Thomas Fan`_.
+
+:mod:`sklearn.utils`
+....................
+
+- |Fix| :func:`utils.check_array` now correctly converts pandas DataFrame with
```

Comment:
does this need to be in both what's news?

Reply:
Yea only in 0.22.1 should be enough.